hangups/conversation.py

```diff
@@ -679,8 +679,7 @@ def next_event(self, event_id, prev=False):
             return self.events[i - 1]
         elif not prev and i + 1 < len(self.events):
             return self.events[i + 1]
-        else:
-            return None
+        return None
```

Comment:
done in https://github.com/tdryer/hangups/pull/343/commits/85e641022d14cea53b0184afa0e64cbcb9e9390c